cfgov/v1/models/ref.py

```diff
@@ -1,21 +1,110 @@
+        ('fed-circuit-court', 'Federal Circuit Court'),
+        ('fed-district-court', 'Federal District Court'),
+        ('state-court', 'State Court'),
+    )),
+    ('Blog', (
+        ('at-the-cfpb', 'At the CFPB'),
+        ('cfpb_report', 'CFPB Report'),
+        ('data-research-reports', 'Data, research & reports'),
+        ('info-for-consumers', 'Info for consumers'),
+    )),
+    ('Enforcement action', (
+        ('fed-district-case', 'Federal District Court Case'),
+        ('admin-adj-process', 'Administrative Adjudication Process'),
+    )),
+    ('Final Rule', (
+        ('interim-final-rule', 'Interim Final Rule'),
+        ('final-rule', 'Final Rule'),
+    )),
+    ('FOIA Frequently Requested Record', (
+        ('report', 'Report'),
+        ('log', 'Log'),
+        ('record', 'Record'),
+    )),
+    ('Implementation Resource', (
+        ('cfpb-bulletins-statements', 'CFPB Bulletins and Statements'),
+        ('impl-compl-material', 'Implementation and Compliance Material'),
+    )),
+    ('Newsroom', (
+        ('blog', 'Blog'),
+        ('op-ed', 'Op-Ed'),
+        ('press-release', 'Press Release'),
+        ('speech', 'Speech'),
+        ('testimony', 'Testimony'),
+    )),
+    ('Notice and Opportunity for Comment', (
+        ('notice-proposed-rule', 'Advanced Notice of Proposed Rulemaking'),
+        ('proposed-rule', 'Proposed Rule'),
+        ('interim-final-rule-2', 'Interim Final Rule'),
+        ('request-comment-info', 'Request for Comment or Information'),
+        ('proposed-policy', 'Proposed Policy'),
+        ('intent-preempt-determ', 'Intent to make Preemption Determination'),
+        ('info-collect-activity', 'Information Collection Activities'),
+        ('notice-privacy-act', 'Notice related to Privacy Act'),
+    )),
+    ('Research Report', (
+        ('consumer-complaint', 'Consumer Complaint'),
+        ('super-highlight', 'Supervisory Highlights'),
+        ('data-point', 'Data Point'),
+        ('snapshot', 'Snapshot'),
+        ('consumer-voices', 'Consumer Voices'),
+        ('education-programs', 'Education and Programs'),
+        ('our-regulations', 'Our Regulations'),
+        ('industry-practices', 'Industry Practices'),
+        ('joint-reports', 'Joint Reports'),
+        ('finances-results', 'Finances and Results'),
+    )),
+    ('Rule under development', (
+        ('notice-proposed-rule-2', 'Advanced Notice of Proposed Rulemaking'),
+        ('proposed-rule-2', 'Proposed Rule'),
+    )),
 ]
+
+page_type_choices = categories + [
+    ('Leadership Calendar', (
+        ('richard-cordray', 'Richard Cordray'),
+        ('meredith-fuchs', 'Meredith Fuchs'),
+        ('steve-antonakes', 'Steve Antonakes'),
+        ('raj-date', 'Raj Date'),
+        ('elizabeth-warren', 'Elizabeth Warren'),
+    ))
+]
+
+
+def choices_for_page_type(page_type):
+    for slug, name in page_types:
+        if page_type == slug:
+            for cat_slug, cat_list in page_type_choices:
```

Comment:
@sebworks This is one of the places where the Leadership Calendar comes into play. Above, in the `page_types`, "Leadership Calendar" is a choice. Since calendars are obviously not categories, I created another list that adds the calendars as choices so it would populate correctly for that page type. There's also some conditional code to set the type [here](https://github.com/cfpb/cfgov-refresh/pull/1314/files#diff-6968a1d9f0cd02ef3900aad4ecc1b09aR79) and [here](https://github.com/cfpb/cfgov-refresh/pull/1314/files#diff-c20b4e892dcf58dabba184f5306e24c1R159) the form uses to change the way they render too.